webpack/assets/javascripts/react_app/components/common/DateTimePicker/DatePicker.js

```diff
@@ -12,13 +12,19 @@ import TodayButton from './DateComponents/TodayButton';
 
 class DatePicker extends React.Component {
   state = {
-    value: this.props.value == null ? new Date() : new Date(this.props.value),
+    value: Date.parse(this.props.value) ? new Date(this.props.value) : new Date(),
+    hiddenValue: this.props.hiddenValue,
   };
   formatDate = () => {
-    const { locale } = this.props;
+    const zeroPadding = n => (n < 10 ? `0${n}` : n);
     const { value } = this.state;
-    const options = { year: 'numeric', month: 'numeric', day: 'numeric' };
-    return value.toLocaleString(locale, options);
+    const date = {
+      year: value.getFullYear(),
+      month: zeroPadding(value.getMonth() + 1),
+      day: zeroPadding(value.getDate()),
+    };
+
+    return `${date.year}-${date.month}-${date.day}`;
```

Comment:
this format isn't affected by the user locale

Reply:
it works exactly the same way as in `DateTimePicker`, I guess this is the format expected on the backend side
https://github.com/theforeman/foreman/blob/5f60fc8f267c232325f54d377e5b5bb2706a4406/webpack/assets/javascripts/react_app/components/common/DateTimePicker/DateTimePicker.js#L38

Reply:
@MariaAga is there a reason for different formats?

Reply:
When I did the dateTimePicker I used locale and it was changed in your PR, not sure why https://github.com/theforeman/foreman/pull/6531

Reply:
Well, I don't remember why, probably because the accepeted format in the server.
Anyway, @kamils-iRonin , can you please extract `formatDate()` to a helper so we keep the code dry?